Sources/StreamVideo/WebSockets/Events/EventNotificationCenter.swift

```diff
@@ -8,7 +8,7 @@ import Foundation
 class EventNotificationCenter: NotificationCenter, @unchecked Sendable {
     private(set) var middlewares: [EventMiddleware] = []
 
-    var eventPostingQueue = DispatchQueue(label: "io.getstream.event-notification-center")
+    var eventPostingQueue = DispatchQueue(label: "io.getstream.event-notification-center", qos: .background)
```

Comment:
not sure if `qos: .background` is suitable here, I would keep the `default` tbh. Can you share your thinking?

Reply:
My thinking is to set low qos to things that are running in the background but that may be too aggressive (in the Tasks implementation that was causing issues). I think you suggestion is right, i'm going to change this to `.default`